src/containers/blocks.jsx

```diff
@@ -460,10 +460,11 @@ class Blocks extends React.Component {
                 />
                 {this.state.prompt ? (
                     <Prompt
+                        defaultValue={this.state.prompt.defaultValue}
                         isStage={vm.runtime.getEditingTarget().isStage}
                         label={this.state.prompt.message}
-                        placeholder={this.state.prompt.defaultValue}
                         showCloudOption={this.state.prompt.showCloudOption}
+                        showMoreOptions={this.state.prompt.showMoreOptions}
```

Comment:
@towerofnix, it looks like something got messed up during the merge conflict resolution. `showMoreOptions` got renamed to `showVariableOptions` a while ago and it looks like you might not have had those changes when you first started implementing this solution.

I think you just need to remove this line..

Reply:
```suggestion
```